.gitignore

```diff
@@ -2,4 +2,7 @@
 .php_cs.cache
 .phpunit.result.cache
 infection.log
+.idea
+
+composer.lock
```

Comment:
Because usually composer.lock require for fixing packages vendors versions. I think it is not reason to fix vendors versions. Especially if you want to override vendors you will meet a problem with fixed composer.lock

Reply:
there is no package required in "require" section, only in "require-dev" so it's not useful to drop it.

Reply:
I had some problem with composer.lock, probably because had  another PHP version than in lock file.

Reply:
Actually GitHub action run with composer install on php 7.2, 7.3, 7.4 and 8.0.

Reply:
@fezfez what do you think about this comment? https://stackoverflow.com/a/12896850/12094762